src/components/exchange/ExchangeAssetList.tsx

```diff
@@ -213,36 +217,44 @@ const ExchangeAssetList = (
         onPress={openVerifiedExplainer}
         scaleTo={0.96}
       >
-        <Header>
+        <Box paddingTop="10px" paddingBottom="2px" paddingLeft="20px">
           <HeaderBackground />
-          <HeaderTitleWrapper>
+          <Box>
             <TitleComponent color={section.color} testID={section.key}>
               {`${section.title}${isVerified ? '  􀅵' : ' '}`}
             </TitleComponent>
-          </HeaderTitleWrapper>
-        </Header>
+          </Box>
+        </Box>
       </ButtonPressAnimation>
     ) : null;
   };
 
-  const FooterSpacer = useCallback(() => (footerSpacer ? <Spacer /> : null), [
-    footerSpacer,
-  ]);
+  const FooterSpacer = useCallback(
+    () => (footerSpacer ? <Box width="full" height={{ custom: 35 }} /> : null),
+    [footerSpacer]
+  );
 
   const isFocused = useIsFocused();
 
   const theme = useTheme();
 
+  type LocalFavourite = Record<string, boolean | undefined>;
```

Comment:
Can we define this inline or outside the component closure? Also might as well match the spelling of the variable 😛 🇺🇸